.gitignore

```diff
@@ -5,4 +5,6 @@ package-lock.json
 .next
 .env.local
 yarn-error.log
-coverage
\ No newline at end of file
+coverage
+*.module.css
+*.module.css.map
```

Comment:
Find out why the module.css.map file is getting created on your machine.

Reply:
I am not sure of the answer yet but for that precaution, I added in gitignore

Reply:
Try to add this code in `next.config.js`  -
```js 
module.exports = {
  // Disable source map generation for CSS modules
  cssModules: {
    generateSourceMaps: false,
  },
};
```
It may or may not work, can you remove the two lines you added to `.gitignore`? I don't think it is important if it is only made in your machine then find the solution for it